### ISSUE TITLE
feature request: lighter comment color

### ISSUE THREAD

**Issue by OneOfOne** [alireza-ahmadi/vscode-theme-gotham#8]
The current comment color is too dark with the dark background, it actually hurts trying to read comments.

Everything else about the theme is perfect.

* using it with Go / TS

**Comment by OneOfOne** [comment]
Currently using this in a personal fork:
```
....
		{
			"name": "Comment",
			"scope": "comment",
			"settings": {
				"foreground": "#13789c"
			}
		},
...
```

**Comment by alireza-ahmadi** [comment]
@OneOfOne This issue has been fixed in #9 and will be available in the next update.
